presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraTestingUtils.java

```diff
@@ -44,7 +44,7 @@ private CassandraTestingUtils() {}
     public static void createTestTables(Cluster cluster, String keyspace, Date date)
     {
         try (Session session = cluster.connect()) {
-            createOrReplaceKeyspace(session, keyspace);
+            createKeyspace(session, keyspace);
         }
         try (Session session = cluster.connect(keyspace)) {
```

Comment:
And catch with `propagate` in case there's a problem with creating a keyspace (due to e.g. name clash). It should never happen but given it's our assertion here, we should not hide this.